V10-BreakingChanges.md

```diff
@@ -1,5 +1,58 @@
-# A collection of updates that change the behaviour
+# A collection of updates that change the behavior
+
+## Async
+
+`parse`, `render` are now async.
```

Comment:
It might be worth having some examples here, since this is probably one of the most important changes!

Reply:
Yes, I'll add documentation soon (with examples and live editor commits for migration)
This file is currently used to keep note of points to elaborate before release.